app/controllers/events_controller.rb

```diff
@@ -0,0 +1,32 @@
+class EventsController < ApplicationController
+  include SessionsHelper
+  include EventsHelper
+
+  def new
+    redirect_to root_path if current_user.nil?
+    @event = Event.new
+  end
+
+  def index
+    @events = Event.all
+  end
```

Comment:
You should take advantage of the scopes you created. This method should list the previous and upcoming events and then you can use them in your views.